requirements/dev.txt

```diff
@@ -5,72 +5,63 @@
 #    make upgrade
 #
 argparse==1.4.0
-astroid==1.5.3
-atomicwrites==1.3.0
-attrs==19.1.0
-backports.functools-lru-cache==1.5
-caniusepython3==7.1.0
-certifi==2019.3.9
+astroid==2.3.3
+backports.functools-lru-cache==1.6.1
+caniusepython3==7.2.0
+certifi==2019.11.28
 chardet==3.0.4
 click-log==0.3.2
 click==7.0
 codecov==2.0.15
-configparser==3.7.4
-contextlib2==0.5.5
-coverage==4.5.3
-diff-cover==2.2.0
-distlib==0.2.9.post0
-django==1.11.21
-edx-lint==1.3.0
-enum34==1.1.6
+coverage==4.5.4
+diff-cover==2.4.1
+distlib==0.3.0
+django==2.2.8
+edx-lint==1.4.1
 filelock==3.0.12
-funcsigs==1.0.2
-futures==3.2.0 ; python_version == "2.7"
```

Comment:
It looks like you ran `make upgrade` under Python 3; until we officially drop Python 2 support, we need to keep running it under 2.7 to pin all the backports that are only needed under that version.

Reply:
I reran it under 2.7.